src/test/server/Server_test.cpp

```diff
@@ -362,12 +357,13 @@ class Server_test : public beast::unit_test::suite
                 thread.get_io_service(), {});
             std::vector<Port> list;
             list.resize(1);
```

Comment:
Same comment here as above - we can remove the `resize` and size the vector in construction.